cmd/soroban-rpc/internal/methods/simulate_transaction.go

```diff
@@ -213,12 +214,16 @@ type SimulateTransactionResponse struct {
 	EventsXDR  []string          `json:"events,omitempty"` // DiagnosticEvent XDR in base64
 	EventsJSON []json.RawMessage `json:"eventsJson,omitempty"`
 
-	MinResourceFee  int64                        `json:"minResourceFee,string,omitempty"`
-	Results         []SimulateHostFunctionResult `json:"results,omitempty"`         // an array of the individual host function call results
-	Cost            SimulateTransactionCost      `json:"cost,omitempty"`            // the effective cpu and memory cost of the invoked transaction execution.
-	RestorePreamble *RestorePreamble             `json:"restorePreamble,omitempty"` // If present, it indicates that a prior RestoreFootprint is required
-	StateChanges    []LedgerEntryChange          `json:"stateChanges,omitempty"`    // If present, it indicates how the state (ledger entries) will change as a result of the transaction execution.
-	LatestLedger    uint32                       `json:"latestLedger"`
+	MinResourceFee int64 `json:"minResourceFee,string,omitempty"`
+	// an array of the individual host function call results
+	Results []SimulateHostFunctionResult `json:"results,omitempty"`
+	// the effective cpu and memory cost of the invoked transaction execution.
+	Cost SimulateTransactionCost `json:"cost,omitempty"`
+	// If present, it indicates that a prior RestoreFootprint is required
+	RestorePreamble *RestorePreamble `json:"restorePreamble,omitempty"`
+	// If present, it indicates how the state (ledger entries) will change as a result of the transaction execution.
+	StateChanges []LedgerEntryChange `json:"stateChanges,omitempty"`
+	LatestLedger uint32              `json:"latestLedger"`
```

Comment:
Stared at this forever and can’t see the diff. Halp?

Reply:
just formatted comments here to keep it above the variable and to reduce total chars per line :|